src/frontends/pytorch/src/op/pad.cpp

```diff
@@ -0,0 +1,110 @@
+        auto step = context.mark_node(opset8::Constant::create(element::i64, Shape{1}, {1}));
+        for (auto i = 0; i < pad_size_half; i++) {
+            ov::NodeVector tensors;
+            pad_r = paddings[pad_last_id - (2 * i + 1)];
+            pad_l = paddings[pad_last_id - (2 * i + 2)];
+            auto axes = context.mark_node(opset8::Constant::create(element::i64, Shape{1}, {2 + i}));
+            if (pad_l > 0) {
+                auto start =
+                    context.mark_node(context.mark_node(opset8::Constant::create(element::i64, Shape{1}, {-pad_l})));
+                auto end = context.mark_node(std::make_shared<opset8::Gather>(
+                    shape,
+                    context.mark_node(opset8::Constant::create(element::i64, Shape{1}, {2 + i})),
+                    context.mark_node(opset8::Constant::create(element::i64, Shape{1}, {0}))));
+
+                auto left = context.mark_node(std::make_shared<opset8::Slice>(cur, start, end, step, axes));
+                tensors.push_back(left);
+            }
+            if (pad_l < 0 || pad_r < 0) {
+                auto start = context.mark_node(
+                    context.mark_node(opset8::Constant::create(element::i64, Shape{1}, {-pad_l > 0 ? -pad_l : 0})));
+                auto end = context.mark_node(
+                    context.mark_node(opset8::Constant::create(element::i64, Shape{1}, {-pad_r > 0 ? pad_r : 0})));
+                auto middle = context.mark_node(std::make_shared<opset8::Slice>(cur, start, end, step, axes));
+                tensors.push_back(middle);
+            } else {
+                tensors.push_back(cur);
+            }
+            if (pad_r > 0) {
+                auto start = context.mark_node(opset8::Constant::create(element::i64, Shape{1}, {0}));
+                auto end = context.mark_node(opset8::Constant::create(element::i64, Shape{1}, {pad_r}));
+                auto right = context.mark_node(std::make_shared<opset8::Slice>(cur, start, end, step, axes));
+                tensors.push_back(right);
+            }
+            if (tensors.size()) {
+                cur = context.mark_node(std::make_shared<opset8::Concat>(tensors, 2 + i));
+            }
+        }
+        return {cur};
+    }
+    if (mode == "constant") {
+        pad_mode = ov::op::PadMode::CONSTANT;
+        if (!context.input_is_none(3)) {
+            value = context.const_input<double>(3);
+        }
+    } else if (mode == "reflect") {
+        pad_mode = ov::op::PadMode::REFLECT;
+    } else if (mode == "replicate") {
+        pad_mode = ov::op::PadMode::EDGE;
+    } else {
+        FRONT_END_OP_CONVERSION_CHECK(false, "aten::pad conversion doesn't support [" + mode + "] padding mode");
+    }
+    return {context.mark_node(std::make_shared<opset8::Pad>(
+        data,
+        pads_begins,
+        pads_ends,
+        context.mark_node(std::make_shared<opset8::Constant>(element::f32, Shape({}), value)),
```

Comment:
Please don't create constant for the case `mode == "constant"`, because `value` is already exist as constant.

Reply:
removed